packages/custom-functions-metadata/test/src/test.ts

```diff
@@ -129,34 +110,7 @@ describe("test errors", function() {
              assert.equal(errtest[2].includes(errorIdBad), true, "Invalid id found");
              assert.equal(errtest[4], `The custom function name "1invalidname" should start with an alphabetic character. (25,19)`);
              assert.equal(errtest[5], `The custom function name "1invalidname" should contain only alphabetic characters, numbers (0-9), period (.), and underscore (_). (25,19)`);
-             assert.strictEqual(fs.existsSync(output), false, "json file created");
-        });
-    });
-});
-describe("test bad file paths", function() {
-    describe("failure to generate bad file path", function() {
-        it("test error file path", async function() {
-            const inputFile = "doesnotexist.ts";
-            const output = "./nofile.json";
-            const testError = "File not found";
-            try {
-                await generate.generate(inputFile, output);
-            } catch (error) {
-                assert.ok(error.message.startsWith(testError), "Error message not found");
-                assert.ok(error.message.includes(inputFile), "File name not found in error message");
-
-            }
-            assert.strictEqual(fs.existsSync(output), false, "json file created");
-        });
-    });
-});
-describe("delete test files", function() {
-    describe("deleting files", function() {
-        it("files to delete", function() {
-            const outputJavaScript = "testjs.json";
-            const outputTypeScript = "test.json";
-            fs.unlinkSync(outputJavaScript);
-            fs.unlinkSync(outputTypeScript);
+             assert.equal(generateResult.metadataJson, "", "should not be any metadata");
```

Comment:
Couldn't we use ```undefined``` instead of the empty ```""``` string?

Reply:
The actual IGenerateResult.metadataJson is a string, and will be empty when there are errors. It's not optional so there is no need to also have it be undefined.